### PR TITLE
docs: add translation

### PR DESCRIPTION
**Pull request by JieDing**
docs:  add translation

### CODE REVIEW

File: cloudevents/translated/zh-cn/spec_CN.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ CloudEvents 是一个用于定义事件格式的供应商中立规范。
 
 ## 文档状态
 
-这份文档是一份仍在制作中的工作草稿。
+这份文档是一份仍在制作中的工作草案。
```
Comment (duglin) [inline cloudevents/translated/zh-cn/spec_CN.md:9]: what changed here? and in the previous edit

Comment (JieDing) [reply cloudevents/translated/zh-cn/spec_CN.md:9]: > previous last two words "草稿" is translated from "draft". But it's not quite formal here. So I use "草案" to replace it.

Comment (duglin) [reply cloudevents/translated/zh-cn/spec_CN.md:9]: ah now I see it - thanks!

**Comment by duglin** [comment]
Approved on the 1/13 call

**Pull request merged**